### ISSUE TITLE
Error: failed to prune cache "assets": remove /opt/build/repo/resources/_gen/assets/js/js/bundle.js no such file or directory

### ISSUE THREAD

**Issue by kimfucious** [Track3/hermit#96]
Hi there,

I'm getting an error when deploying my Hermit themed Hugo site to Netlify, and I'm thinking this is theme related and not Netlify related, but I could be entirely wrong.

I've installed Hermit as a submodule, and when I deploy, I get this error:

Where might I begin to start looking for ideas on how to resolve this?

Thanks.

```console
5:44:05 PM: Error: failed to prune cache "assets": remove /opt/build/repo/resources/_gen/assets/js/js/bundle.js_d11fe7b62c27961c87ecd0f2490357b9.content: no such file or directory
```

**Comment by kimfucious** [comment]
Okay, after some trial and error, I've discovered that I can reproduce this locally on my dev box.

And, I can get this to build if I use: `hugo --minify`, omitting the `--gc`

Not sure if this is best practice. Thoughts?

**Comment by jzeneto** [comment]
Relates to gohugoio/hugo#5745

**Comment by Track3** [comment]
Hi all, this is fixed in Hugo 0.58.2, close this issue for now. 🎉
